### PR TITLE
Bug 1206471 - Disable cellular data options when no SIM, r=gasolin

### PR DESCRIPTION
**Pull request by scottwu**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d334366ead7907c9e241b4ccf9ee9894c7778c8e

File: apps/settings/js/carrier.js [hunk 1]
```diff
@@ -18,6 +18,25 @@ var CarrierSettings = (function() {
   /** Flag */
   var _restartingDataConnection = false;
 
+  var _elements = {
+    detailHeader:
```
Comment (gasolin) [inline apps/settings/js/carrier.js]: thanks for the refactor :+1:

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7121e14c3c2b7b9e011dd10ca6e21e670dfeb813

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d7756cfc8e8126ef9294418c6997d632d6858b8f

**Pull request merged**